### ISSUE TITLE
Suggestion - apply opacity to images

### ISSUE THREAD

**Issue by jzhu01** [ionpattern/prj-rev-bwfs-tea-cozy#3]
Seen here: https://github.com/ionpattern/prj-rev-bwfs-tea-cozy/blob/master/TeaCozy/resources/css/style.css#L18

Since the text is already on a black background, it's going to be harder to read it with the opacity. I recommend instead applying this opacity to the images, so it becomes easier to read the overlaying text.

**Comment by ionpattern** [comment]
I can see how to apply opacity to ordinary images (by applying to img), but what about background images?

Also just FYI the design spec was very confusing with regard to opacity, since it didn't suggest where to apply it.

**Comment by jzhu01** [comment]
Yes, good point! In the case of background images, it might be better to leave it off. Sorry for the delay in responding, I take awhile. :sweat_smile: 

Here's a link if you're curious about how to do it: https://css-tricks.com/snippets/css/transparent-background-images/

I agree, the spec was pretty vague, which is why I made this as a suggestion. Wish you the best! :)
